tests/test_cli.py

```diff
@@ -286,18 +289,18 @@ def test_fix(tmp_path: Path) -> None:
     assert fixed_notebook.nbformat == notebook_1.nbformat
     assert fixed_notebook.nbformat_minor == notebook_1.nbformat_minor
     assert fixed_notebook.cells == notebook_1.cells + [
-        BaseCell(
+        MarkdownCell(
             metadata=CellMetadata(git_hash=id_main),
             source=[f"`<<<<<<< {id_main}`"],
             cell_type="markdown",
```

Comment:
Pedantry: do we still need this with the better cell type?
(same for below)

Reply:
Do you mean the `cell_type` argument? If so, then no, we don't need this anymore with a better (more concrete) cell type.

Reply:
Alright sounds good! That's better indeed - can we drop that?